plugins/modules/nfs.py

```diff
@@ -196,6 +129,77 @@
             type: str
             choices: [absent, present]
             default: present
+      user:
+        description:
+        - Specifies the persona name.
+        type: str
+  path:
+    description:
+    - Specifies the filesystem path. It is the absolute path for System access zone
+      and it is relative if using non-system access zone.
+    - For example, if your access zone is 'Ansible' and it has a base path
+      '/ifs/ansible' and the path specified is '/user1', then the effective
+      path would be '/ifs/ansible/user1'.
+    - If your access zone is System, and you have 'directory1' in the access
+      zone, the path provided should be '/ifs/directory1'.
+    - The directory on the path must exist - the NFS module will not create
+      the directory.
+    - Ansible module will only support exports with a unique path.
+    - If there are multiple exports present with the same path, fetching details,
+      creation, modification or deletion of such exports will fail.
+    required: true
+    type: str
+  read_only:
+    description:
+    - Specifies whether the export is read-only or read-write. This parameter
+      only has effect on the 'clients' list and not the other three types of
+      clients.
+    - This setting can be modified any time. If it is not set at the time of
+      creation, the export will be of type read/write.
+    type: bool
+  read_only_clients:
+    description:
+    - Specifies the clients with read-only access to the export, even when the
+      export is read/write.
+    - This list can be changed anytime during the lifetime of the NFS export.
+    - If the read_only_clients list needs to be replaced, C(client_state) should not be provided.
```

Comment:
```suggestion
    - If I(client_state) is not provided, then the host machine will replicate the values provided in the I(read_only_clients).
 ```
The same can be updated in other description as well